src/classes/Booking.js

```diff
@@ -0,0 +1,14 @@
+import Room from './Room.js';
+import Customer from './Customer.js'
```

Comment:
I'm curious why you're importing your room and customer classes in here...are you planning on instantiating them from within the booking class?

Reply:
I was initially - I tried to set the classes up in anticipation of the functionality I might need. Now that I'm farther into the project I see that it wasn't necessary. I'll get rid of those.